scripts/data_deletion/delete_training_data_utils.js

```diff
@@ -207,7 +205,7 @@ var filterFamilyMembers = function(personsList, logdir) {
 
 // Find which facilities (if any) this person is a contact for.
 var isContactFor = function(db, personId) {
-  return db.query('medic-client/places_by_contact', {key: [personId], include_docs: true})
```

Comment:
Right I understand that medic-client is only available on the browser/client but still seems interesting how we decide when to move one to another.